### ISSUE TITLE
Emmet prediction's pane is Blank

### ISSUE THREAD

**Issue by Elie77350** [microsoft/vscode#153307]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:
- OS Version:
![Capture d’écran 2022-06-27 à 15 17 23](https://user-images.githubusercontent.com/75457081/175952226-037359e7-3eae-4cf0-99d8-f35d612b762d.png)

Steps to Reproduce:

1. 1.68.1
2. macOS Monterey 12.4

**Comment by Elie77350** [comment]
Does this issue occur when all extensions are disabled?: Yes/No

I just realized that I hadn't answered the question, I just tested, the problem is still present even when all the plugins are deactivated

**Comment by rzhao271** [comment]
I'm curious whether this issue will appear after you upgrade VS Code Stable. If it doesn't happen on Insiders, it shouldn't happen on Stable, but if it does still happen on Stable after the next version, there might be some settings causing that behaviour.

**Comment by Elie77350** [comment]
Hello, I will look at it very closely at the time of the next update. I will keep you informed as soon as possible.

**Comment by VSCodeTriageBot** [comment]
Hey @rzhao271, this issue might need further attention.

@Elie77350, you can help us out by closing this issue if the problem no longer exists, or adding more information.

**Comment by Elie77350** [comment]
Hello @rzhao271,

hello, unfortunately the problem appeared again just after the last update of vscode 
Version: 1.71.2 (Universal)


![bug on intelisence in vscode](https://user-images.githubusercontent.com/75457081/190932430-a90dbf14-2f80-4881-9ecd-b3c043772c7b.png)

**Comment by rzhao271** [comment]
I don't think those icons correspond to Emmet completions. There's something wrong with the completions pane in general, and the rendering looks garbled there.

Does running the program from the terminal with `code --disable-gpu` work?

**Comment by Elie77350** [comment]
hello,
 I just tested, with this command ( code --disable-gpu) but there is no improvement!
autocompletion works and Emmet too!
But still no display.

any other similar cases since the last update?

**Comment by Elie77350** [comment]
> I don't think those icons correspond to Emmet completions. There's something wrong with the completions pane in general, and the rendering looks garbled there.
> 
> Does running the program from the terminal with `code --disable-gpu` work?

Hello, rzhao271

I found the time to look for the cause of Emmet's malfunction, with each update.
I went to the JSON file, and started commenting out one by one, all of my vscode's sync settings.

it was when commenting ("editor.suggestFontSize": 1, ), that Emmet started working correctly again.
![Capture d’écran 2022-10-13 à 02 41 19](https://user-images.githubusercontent.com/75457081/195473979-afa30758-4ad6-4796-a01e-03776a04d2d2.png)
![Capture d’écran 2022-10-13 à 02 43 05](https://user-images.githubusercontent.com/75457081/195473989-1a4fe46a-7609-4fb2-b716-28f170b87201.png)

Hello,

I found the time to look for the cause of Emmet's malfunction, with each update.
I went to the JSON file, and started commenting out one by one, all of my vscode's sync settings.
it was when commenting ("editor.suggestFontSize": 1, ), that Emmet started working correctly again.
So we can close this issue. The reason why, the issue was showing up with every update, is that I was syncing my data in vscode with my GitHub account. So I retrieved a parameter error hosted in the cloud and I repatriated it to my machine.

Thanks again,

The last. mse update from VsCode is awesome!
